dev_local_env/files/macosx/requirements.txt

```diff
@@ -0,0 +1,11 @@
+ansible==2.7.5    
```

Comment:
All these versions need to be bumped (as EOL in general). 

Because of current incompatibilities with non-EOL Ansible and mitogen, there's a 0.3.0cf1 version I built from a PR that supports Ansible 4.7.0. However, I can't expect every team to have access to our CodeArtifact so we should also just remove the mitogen dependency and all steps associated. Can add a comment specifying where to look in Confluence instead.

Reply:
Thanks yeah I guess I'll go ahead and merge for now.  And then we can PR version changes/comments later.